### DISCUSSION TITLE
Any way to archive tasks individually?

### DISCUSSION THREAD

**Discussion by skostojohn** [ransome1/sleek#708] [Ideas]
Just started using sleek and I realize I would like to archive older completed tasks, but not recently completed ones (Which I often need to make active again).  Archiving appears to be an all-or-none operation.  It would be helpful to be able to either archive tasks one by one, or archive anything that has not been touched in some time  period (e.g. 7 days).

Does anyone else feel this would be a useful feature?

**Comment by fccfc5d245c** [comment]
Good idea, @skostojohn , thank you , I've found myself wanting that feature many times: 
Some completed tasks I need to keep close, to track my progress on ongoing projects or to recall something (completed tasks can contain relevant info for future tasks like a URL I'll use again or something like "cut a 140mm piece for the build", that measurement will come up again). 
But other completed tasks (sweep the floor after or the boss will nag) I just want to complete and forget about them, they only add clutter if I don't archive Everything.

A button to archive the selected task, maybe to the right of the Pomodoro option or on the left-click menu, would do it for me.
